src/components/gas/GasSpeedButton.js

```diff
@@ -107,13 +109,16 @@ const GasSpeedPagerCentered = styled(Centered).attrs(() => ({
 
 const TextContainer = styled(Column).attrs(() => ({}))({});
 
-const TransactionTimeLabel = ({ formatter, theme }) => {
+const TransactionTimeLabel = ({ formatter, isLongWait, theme }) => {
   const { colors } = useTheme();
   return (
     <Label
       align="right"
+      // eslint-disable-next-line no-nested-ternary
       color={
-        theme === 'dark'
+        isLongWait
+          ? colors.lightOrange
+          : theme === 'dark'
```

Comment:
😅  wdyt of moving `color` definition above the return so we don't ignore the warning?